etna/records/templatetags/records_tags.py

```diff
@@ -9,3 +10,9 @@
 def is_page_current_item_in_hierarchy(page: Record, hierarchy_item: dict):
     """Checks whether given page matches item from a record's hierarchy"""
     return page.reference_number == hierarchy_item["reference_number"]
+
+
+@register.filter
+def as_label(record_field_name: str) -> str:
+    """returns human readable pre configured record/api field name, otherwise blank"""
+    return human_readable_labels.get(record_field_name, "")
```

Comment:
- If someone forgets to update the mapping it should go noticed as "UNRECOGNISED FIELD NAME", so that it need to be configured for a Human readable label.
- Not converting a field name to sentence case or striping underscore, for an unconfigured field name as this will still retain the non-human name.

Reply:
As discussed updated default to "UNRECOGNISED FIELD NAME"